src/cmd.ts

```diff
@@ -245,6 +245,12 @@ function createCommandWithSharedOptions(name, description) {
 			"The number of searches before stops",
 			parseInt,
 			fallback(fileConfig.searchLimit, 0)
+		)
+		.option(
+			"--block-list <strings>",
+			"The strings you wish to block",
```

Comment:
can we be clearer about what gets matched against these strings? i.e. torrent/release names?

Reply:
We could use a simple (enough) regex to match for infohash and separate the searching so we don't run through needlessly.

Yes i can improve the messaging.